lib/commands/command-latest.bash

```diff
@@ -5,18 +5,31 @@ latest_command() {
 
   local plugin_name=$1
   local query=$2
+  local plugin_path
 
   [[ -z $query ]] && query="$DEFAULT_QUERY"
 
+  plugin_path=$(get_plugin_path "$plugin_name")
+  check_if_plugin_exists "$plugin_name"
+
   local versions
-  # pattern from xxenv-latest (https://github.com/momo-lab/xxenv-latest)
-  versions=$(asdf list-all "$plugin_name" "$query" |
-    grep -vE "(^Available versions:|-src|-dev|-latest|-stm|[-\\.]rc|-alpha|-beta|[-\\.]pre|-next|(a|b|c)[0-9]+|snapshot|master)" |
-    sed 's/^\s\+//' |
-    tail -1)
-
-  if [ -z "${versions}" ]; then
-    exit 1
+
+  if [ -f "${plugin_path}/bin/latest-stable" ]; then
+    versions=$(bash "${plugin_path}"/bin/latest-stable "$query")
```

Comment:
FYI at some point hopefully in the near future I'd like to explore allowing callbacks to be written in any language and not just bash. We'd need to change the callback invocations from `bash "${plugin_path}"/bin/<callback> <args>` to just `"${plugin_path}"/bin/<callback> <args>`. This will give plugin creators more control over how they implement callbacks.

Reply:
Sure! It sounds good. I don't mind to look into it when the time comes.